# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/systemservices/android_video_capture/video_capture_android.h
# Commit: 2796c6521f19
# Full Hash: 2796c6521f198e7dc0e36b2ca02ceb9ba01da143
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2020-02-13 18:21:33
# Regressor Bug: 1578073
# File Overlap Count: 1
# Description:
#   Bug 1578073 - Fix include paths for android video capture; r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D61857
# ==============================================================================

diff -r 7d9c51a90036 -r 2796c6521f19 dom/media/systemservices/android_video_capture/video_capture_android.h
--- a/dom/media/systemservices/android_video_capture/video_capture_android.h	Tue Feb 11 17:06:09 2020 +0000
+++ b/dom/media/systemservices/android_video_capture/video_capture_android.h	Tue Feb 11 17:06:37 2020 +0000
@@ -13,7 +13,7 @@
 
 #include <jni.h>
 
-#include "modules/video_capture/android/device_info_android.h"
+#include "device_info_android.h"
 #include "modules/video_capture/video_capture_impl.h"
 
 namespace webrtc {
